internal/proto/local/controller/api/services/v1/managed_group_service.proto

```diff
@@ -0,0 +1,129 @@
+syntax = "proto3";
+
+package controller.api.services.v1;
+
+option go_package = "github.com/hashicorp/boundary/internal/gen/controller/api/services;services";
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/wrappers.proto";
+import "controller/api/resources/managedgroups/v1/managed_group.proto";
+
+service ManagedGroupService {
+
+  // GetManagedGroup returns a stored ManagedGroup if present. The provided request must
+  // include the id for the ManagedGroup be retrieved. If missing, malformed or
+  // referencing a non existing ManagedGroup an error is returned.
+  rpc GetManagedGroup(GetManagedGroupRequest) returns (GetManagedGroupResponse) {
+    option (google.api.http) = {
+        get: "/v1/managed-groups/{id}"
+        response_body: "item"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Gets a single ManagedGroup."
+    };
+  }
+
+  // ListManagedGroups returns a list of stored ManagedGroups which exist inside the
+  // provided Auth Method. The request must include the Auth Method id which
+  // contains the ManagedGroups being listed. If missing or malformed an error
+  // is returned.
+  rpc ListManagedGroups(ListManagedGroupsRequest) returns (ListManagedGroupsResponse) {
+    option (google.api.http) = {
+      get: "/v1/managed-groups"
+     };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Lists all ManagedGroups in a specific Auth Method."
+    };
+  }
+
+  // CreateManagedGroup creates and stores an ManagedGroup in boundary. The provided
```

Comment:
Nit: capitalize "Boundary" and I'm not sure the phrase "in Boundary" is needed anywho.

Reply:
I removed it